controllers/host/host_controller.go

```diff
@@ -249,19 +249,6 @@ func (r *HostReconciler) UpdateRequired(instance *starlingxv1.Host, profile *sta
 		opts.Personality = profile.Personality
 	}
 
-	if profile.SubFunctions != nil {
```

Comment:
the subfunctions can't be modified after initial provisioning . 
And if they are set by the install menus anyway 

host-update doesn't need the subfunctions to be specified
info is recieved from /etc/platform/platform.conf 

```
sysadmin@controller-0:~$ cat /etc/platform/platform.conf
nodetype=controller
subfunction=controller,worker
system_type=All-in-one
http_port=8080
management_interface=enp0s8
INSTALL_UUID=16a32937-2c2c-49e5-8e9f-295ade9a83e3
UUID=4c57a190-dea5-40e2-9469-2f9123def582
sw_version=23.09
vswitch_type=none
system_mode=duplex
sdn_enabled=no
region_config=no
security_feature="nopti nospectre_v2 nospectre_v1"
cluster_host_interface=enp0s8
oam_interface=enp0s3

```

Reply:
After removing the subfunction reconcile code the kernel changes can be modified 
standard -> lowlatency -> standard without sysinv rest api rejecting the subfunction patch

Reply:
Removing this part means we don't support subfunction configuration in bootstrap via configuration yaml?
Also what is the insync status after bootstrap and day2 operation? Does it become true?

Reply:
After our online discussion. I was goind to remove subfunctions from the CRDs but the "worker" subfunction is used everywhere in the logic. I will attempt to create some if the logic around lowlatency and the kernel